src/entities/risk-flag.entity.ts

```
@@ -26,6 +26,12 @@ export class RiskFlag {
   @Column({ type: 'text', nullable: true })
   suggestedText: string;
 
+  @Column({ type: 'text', nullable: true })
+  notes: string;
+
+  @Column({ enum: ['open', 'resolved', 'ignored'], default: 'open', type: 'enum' })
```

ESLint

[error] Replace `·enum:·['open',·'resolved',·'ignored'],·default:·'open',·type:·'enum'` with `⏎····enum:·['open',·'resolved',·'ignored'],⏎····default:·'open',⏎····type:·'enum',⏎·`

(prettier/prettier)